docs/faq.md

```diff
@@ -193,3 +193,12 @@ To unload the model and free up memory use:
 ```shell
 curl http://localhost:11434/api/generate -d '{"model": "llama2", "keep_alive": 0}'
 ```
+
+## Controlling which GPUs to use
+
+By default, on Linux and Windows, Ollama will attempt to use Nvidia GPUs, or
+Radeon GPUs, and will use all the GPUs it can find. You can limit which GPUs
+will be utilized by setting the environment variable `CUDA_VISIBLE_DEVICES` for
+NVIDIA cards, or `HIP_VISIBLE_DEVICES` for Radeon GPUs to a comma delimited list
+of GPU IDs. You can set to an invalid GPU ID to bypass the GPU and fallback to
+CPU.
```

Comment:
What is an invalid GPU ID,`-1`? I think it would be clearer to specify that they can set it to `-1` to bypass GPUs